src/components/Checkbox/CheckBoxIcon.js

```diff
@@ -0,0 +1,57 @@
+import cx from 'classnames';
+import React from 'react';
+import PropTypes from 'prop-types';
+import { iconChevronDown } from 'carbon-icons';
+import Icon from '../Icon';
+
+const groupArrow = {
```

Comment:
This looks like it might be problematic for responsive and/or style overrides. Perhaps Flexbox could be used instead of a static value? We might need to dig in and figure out how we get default styles into these controls - there has to be a way they do it internally.